### DISCUSSION TITLE
Codespace wont load

### DISCUSSION THREAD

**Discussion by Brougud** [community/community#7921] [Codespaces]
When I go into the codespace of a repository, it will just sit there and load. It will keep loading and wont show the files that are there. I am upset because this was the best way I can find for me to code on devices that I can not use Visual studio code on.

**Comment by n123456789k** [comment]
Same issue here; https://github.com/github/feedback/discussions/7920

**Comment by asciimike** [comment]
Hey folks, sorry you're having issues with Codespaces performance. We're actively investigating these issues and will get them resolved as quickly as possible!
